rlberry/manager/agent_manager.py

```diff
@@ -470,21 +472,29 @@ def get_agent_instances(self):
             ]
         return []
 
-    def eval_agents(self, n_simulations: Optional[int] = None) -> list:
+    def eval_agents(
+        self,
+        n_simulations: Optional[int] = None,
+        eval_kwargs: Optional[dict] = None,
+    ) -> list:
```

Comment:
Another minor comment about the typing: I think it's probably better to precise also the underlying type of list, e.g. list[int] for [1, 2, 4]